lib/core/notifiers/report_attendance_notifier.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:ata/core/notifiers/base_notifier.dart';
+import 'package:ata/core/services/user_service.dart';
+import 'package:ata/util.dart';
+class ReportAttendanceNotifier extends BaseNotifier {
+  final UserService _userService;
+
+  ReportAttendanceNotifier(UserService userService) : _userService = userService;
+
+  Map recordAttendanceList = new Map();
+ 
+  //* all async request here
+  Future<void> refresh() async {
+    setBusy(true);
+    recordAttendanceList = await fetchDataRecordAttendance();
+    setBusy(false);
+  }
+
+  Future fetchDataRecordAttendance() async {
```

Comment:
fetchDataRecordAttendance -> pls put this to UserService, need to handle responseData  here
please create a "**class**" to hold Attendance record